### ISSUE TITLE
Incompatible with the built-in `@profview` of VSCode

### ISSUE THREAD

**Issue by taketo1024** [tkluck/GaloisFields.jl#23]
`GaloisFields` seems to be incompatible with the built-in `@profview` of VSCode. 
When I run the following simple code:

```julia
using GaloisFields
@profview println("hello")
```

I get the error:

```
ERROR: LoadError: UndefVarError: F not defined
Stacktrace:
  [1] Base.Broadcast.BroadcastStyle(T::Type{Tuple{}})
    @ GaloisFields.Broadcast ~/.julia/packages/GaloisFields/41126/src/Broadcast.jl:59
  [2] combine_styles(c::Tuple{})
    @ Base.Broadcast ./broadcast.jl:435
  [3] broadcasted(::Function, ::Tuple{})
    @ Base.Broadcast ./broadcast.jl:1298
  [4] var"@profview"(__source__::LineNumberNode, __module__::Module, ex::Any, args::Vararg{Any})
    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.7.5/scripts/packages/VSCodeServer/src/profiler.jl:132
  [5] eval
    @ ./boot.jl:368 [inlined]
  [6] include_string(mapexpr::typeof(identity), mod::Module, code::String, filename::String)
    @ Base ./loading.jl:1428
  [7] include_string(m::Module, txt::String, fname::String)
    @ Base ./loading.jl:1438
  [8] invokelatest(::Any, ::Any, ::Vararg{Any}; kwargs::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Base ./essentials.jl:729
  [9] invokelatest(::Any, ::Any, ::Vararg{Any})
    @ Base ./essentials.jl:726
 [10] inlineeval(m::Module, code::String, code_line::Int64, code_column::Int64, file::String; softscope::Bool)
    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.7.5/scripts/packages/VSCodeServer/src/eval.jl:233
 [11] (::VSCodeServer.var"#66#70"{Bool, Bool, Bool, Module, String, Int64, Int64, String, VSCodeServer.ReplRunCodeRequestParams})()
    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.7.5/scripts/packages/VSCodeServer/src/eval.jl:157
 [12] withpath(f::VSCodeServer.var"#66#70"{Bool, Bool, Bool, Module, String, Int64, Int64, String, VSCodeServer.ReplRunCodeRequestParams}, path::String)
    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.7.5/scripts/packages/VSCodeServer/src/repl.jl:249
 [13] (::VSCodeServer.var"#65#69"{Bool, Bool, Bool, Module, String, Int64, Int64, String, VSCodeServer.ReplRunCodeRequestParams})()
    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.7.5/scripts/packages/VSCodeServer/src/eval.jl:155
 [14] hideprompt(f::VSCodeServer.var"#65#69"{Bool, Bool, Bool, Module, String, Int64, Int64, String, VSCodeServer.ReplRunCodeRequestParams})
    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.7.5/scripts/packages/VSCodeServer/src/repl.jl:38
 [15] (::VSCodeServer.var"#64#68"{Bool, Bool, Bool, Module, String, Int64, Int64, String, VSCodeServer.ReplRunCodeRequestParams})()
    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.7.5/scripts/packages/VSCodeServer/src/eval.jl:126
 [16] with_logstate(f::Function, logstate::Any)
    @ Base.CoreLogging ./logging.jl:511
 [17] with_logger
    @ ./logging.jl:623 [inlined]
 [18] (::VSCodeServer.var"#63#67"{VSCodeServer.ReplRunCodeRequestParams})()
    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.7.5/scripts/packages/VSCodeServer/src/eval.jl:225
 [19] #invokelatest#2
    @ ./essentials.jl:729 [inlined]
 [20] invokelatest(::Any)
    @ Base ./essentials.jl:726
 [21] macro expansion
    @ ~/.vscode/extensions/julialang.language-julia-1.7.5/scripts/packages/VSCodeServer/src/eval.jl:34 [inlined]
 [22] (::VSCodeServer.var"#61#62")()
    @ VSCodeServer ./task.jl:484
```

I'd be glad if you could tell me a workaround for this issue.

**Comment by tkluck** [comment]
Thanks for reporting! I think I just fixed this and I'll push a new release.

**Comment by tkluck** [comment]
Fixed in v1.2.1, to which you can upgrade when https://github.com/JuliaRegistries/General/pull/66991 is merged.

**Comment by taketo1024** [comment]
Thank you very much! 
I have confirmed that `@profview` works fine in v1.2.1.
